custom_components/tuya_local/devices/moes_wht009_thermostat.yaml

```diff
@@ -0,0 +1,226 @@
+        range:
+          min: 150
+          max: 450
+        mapping:
+          - scale: 10
+            step: 10
+  - entity: sensor
+    name: Current temperature
+    class: temperature
+    dps:
+      - id: 24
+        type: integer
+        name: sensor
+        class: measurement
+        unit: C
+        mapping:
+          - scale: 10
+            step: 5
+  - entity: number
+    name: Minimum temperature
+    class: temperature
+    category: config
+    icon: "mdi:thermometer-chevron-down"
+    dps:
+      - id: 26
+        type: integer
+        name: value
+        optional: true
+        unit: C
+        range:
+          min: 50
+          max: 150
+        mapping:
+          - scale: 10
+            step: 10
+  - entity: number
+    name: Temperature correction
+    category: config
+    icon: "mdi:thermometer-check"
+    dps:
+      - id: 27
+        type: integer
+        optional: true
+        name: value
+        range:
+          min: -9
+          max: 9
+  - entity: sensor
+    name: Hvac action
+    dps:
+      - id: 36
+        type: string
+        name: sensor
+  - entity: lock
+    translation_key: child_lock
+    category: config
+    dps:
+      - id: 40
+        type: boolean
+        name: lock
+  - entity: sensor
+    name: External temperature
+    class: temperature
+    dps:
+      - id: 101
+        type: integer
+        name: sensor
+        class: measurement
+        unit: C
+        mapping:
+          - scale: 10
+            step: 5
+  - entity: select
+    name: Temperature sensor type
+    icon: "mdi:home-thermometer"
+    category: config
+    dps:
+      - id: 102
+        type: string
+        optional: true
+        name: option
+        mapping:
+          - dps_val: IN
+            value: Internal
+          - dps_val: OU
+            value: External
+          - dps_val: AL
+            value: Both
+  - entity: number
+    name: Temperature hysteresis
+    category: config
+    icon: "mdi:thermometer-plus"
+    dps:
+      - id: 103
+        type: integer
+        optional: true
+        name: value
+        unit: °
+        range:
+          min: 1
+          max: 5
+  - entity: number
+    name: High protection temperature setting
+    class: temperature
+    category: config
+    icon: "mdi:thermometer-chevron-up"
+    dps:
+      - id: 104
+        type: integer
+        name: value
+        optional: true
+        unit: C
+        range:
+          min: 100
+          max: 700
+        mapping:
+          - scale: 10
+            step: 10
+  - entity: number
+    name: Low protection temperature setting
+    class: temperature
+    category: config
+    icon: "mdi:thermometer-chevron-down"
+    dps:
+      - id: 105
+        type: integer
+        name: value
+        optional: true
+        unit: C
+        range:
+          min: 0
+          max: 100
+        mapping:
+          - scale: 10
+            step: 10
+  - entity: number
+    name: Eco mode temperature setting
```

Comment:
The last three names seem a bit long. This one can be "Eco temperature", I am not sure about the two previous ones, as the meaning is not exactly clear, maybe Anti-frost temperature for the low one, if I am understanding it correctly, but I am not sure about what the high one is for.